src/content/docs/en/guides/cms/decap-cms.mdx

```diff
@@ -20,7 +20,7 @@ i18nReady: false
 
 ## Production Sites
 
-The following sites use Astro + Netlify CMS in production:
+The following sites use Astro + Decap CMS in production:
```

Comment:
```suggestion
The following sites use Astro + Netlify CMS in production:
```
I checked the source code, and this project still appears to be using Netlify CMS